layouts/home.html

```diff
@@ -72,111 +72,107 @@ <h1 class="text-5xl font-medium">{{ .Title }}</h1>
+	  "ButtonLink" "./integrate/riot/"
+          "ButtonLabel" "Redis input and output CLI tool user guide"  
           "LinksLeft" (slice
-            (dict "Text" "Install" "URL" "./operate/redisinsight/install/")
-            (dict "Text" "User guide" "URL" "./develop/tools/insight/")
-            (dict "Text" "Manage Streams in Redis Insight" "URL" "./develop/tools/insight/tutorials/insight-stream-consumer/")
+            (dict "Text" "Redis Input/Output Tool quick start" "URL" "./integrate/riot/quick-start/")
           )
         ) }}
+	{{ partial "docs-section.html" (dict
+	  "Title" "Connect with Redis client API libraries"
+          "Description" "Connect your application to a Redis database and try an example"
+          "ButtonLink" "./develop/clients/"
+          "ButtonLabel" "Redis client API library guides"
+       
+          "LinksLeft" (slice
+            (dict "Text" "Python" "URL" "./develop/clients/redis-py/")
+	    (dict "Text" "C#/.NET" "URL" "./develop/clients/dotnet/")
+	    (dict "Text" "JavaScript" "URL" "./develop/clients/nodejs/")
+            (dict "Text" "Java" "URL" "./develop/clients/jedis/")
+            (dict "Text" "Go" "URL" "./develop/clients/go/")
+            (dict "Text" "PHP" "URL" "./develop/clients/php/")
+          )  ) }}
       </div>
       <div class="lg:px-11 w-full flex flex-col gap-3">
-        <h2 class="text-5xl font-medium" id="operate">Explore</h2>
+        <h2 class="text-5xl font-medium" id="operate">Deploy</h2>
         {{ partial "docs-section.html" (dict
-          "Title" "Redis and Stack"
-          "Description" "Redis Stack extends Redis with modern data models and processing engines."
-          "ButtonLink" "./operate/oss_and_stack/"
-          "ButtonLabel" "Read more"
-          "LinksLeftTitle" "Learn more"
+            "Title" "Deploy Redis clusters for scale and availability"
+          "Description" "Robust support for hybrid and geo-distributed environments"
+	  "ButtonLink" "./operate/rs/installing-upgrading/install/plan-deployment/"
+          "ButtonLabel" "Redis deployment planning"  
           "LinksLeft" (slice
-            (dict "Text" "Install" "URL" "./operate/oss_and_stack/install/install-stack/")
-            (dict "Text" "Redis Query Engine" "URL" "./develop/interact/search-and-query/")
-            (dict "Text" "JSON" "URL" "./develop/data-types/json/")
+           (dict "Text" "Optimize Redis clusters guide" "URL" "./operate/rs/clusters/optimize")
           )
         ) }}
         {{ partial "docs-section.html" (dict
-          "Title" "Redis Cloud"
-          "Description" "Deploy Redis Enterprise on Amazon Web Services, Google Cloud, or Microsoft Azure."
-          "ButtonLink" "./operate/rc"
-          "ButtonLabel" "Read more"
-          "LinksLeftTitle" "Learn more"
+           "Title" "Deploy on containers running in Kubernetes"
+          "Description" "Get started with the Redis Software for Kubernetes container image"
```

Comment:
```suggestion
          "Description" "Get started with the Redis for Kubernetes"
```
Container image isn't really the best way to describe this. While K8s does use containers, they are orchestrated by K8s and the user actually interacts with the "deployment" resource, not the container.

Reply:
Thanks! I changed the button text, but I want to keep the words container and image on this page for SEO for now.